### DISCUSSION TITLE
Differences between factory and "plain" bin

### DISCUSSION THREAD

**Discussion by dat-master** [benzino77/tasmocompiler#261] [Q&A] [answered]
This is a great tool, must be the fastest and easiest way to produce Tasmota firmware.
I did a couple of builds and noticed that only the firmware.factory.bin works, the "plain" firmware.bin does not.
I use the same command line to burn both bins e.g. 
esptool.py -p /dev/ttyUSB0 -b 460800 write_flash 0x00000 ~/downloads/firmware.bin
Do I need to change the address or use some other parameter for firmware.bin to work?
What are the differences between them?
I am programming a module with an ESP32-D0WDQ6 (revision 1) chip with binaries produced by https://gitpod.io/#https://github.com/benzino77/tasmocompiler
Thanks

**Comment by benzino77** [comment, marked as answer]
Hi,

Have a look at discord channel, there was [couple ](https://discord.com/channels/479389167382691863/790187952416358460/948984886054776843) of questions regarding those binaries. Plain binary has to be [flashed ](https://tasmota.github.io/docs/ESP32/#flashing) with `partitions` , `bootloaders` and `boot_app0` files (three additional files needed), using `factory` binary you only need this one.
This is only my understanding, for more information you should go to Tasmota Discord Channel and ask Tasmota Dev Team.

**Reply by dat-master** [reply]
Thanks for the prompt reply.
I suspected something like that.
Didn't know about the discord channel, I will go there too.
